usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -35,9 +35,11 @@ SELECT
   UTM.award_category,
   UTM.recipient_unique_id,
   UTM.parent_recipient_unique_id,
+  CONCAT(UTM.recipient_hash, '-', case when UTM.parent_recipient_unique_id is null then 'R' else 'C' end) as recipient_hash,
   UTM.recipient_name,
 
   UTM.action_date,
+  DATE(UTM.action_date + interval '3 months') AS fiscal_action_date,
```

Comment:
This is just to get the fiscal year, right?

Reply:
Not just fiscal year. For spending over time you need the month, day, and year as it relates to the fiscal year. Instead of splitting them up into three different components you can just add three months to the action_date and then use the date_histogram aggregation on elasticsearch.

Reply:
Hmm. I don't have any productive ideas, but the concept of a "fiscal date" where today is December 11 in but simultaneously sometime in March in fiscal world seems like a good way to confuse a future dev. Just putting it out there.

Reply:
Oh I agree that it is weird, but that is current functionality for spending_over_time with the FiscalYear, FiscalMonth, and FiscalQuarter being the current month + 3 months since the fiscal year runs Oct 1 - Sept 30.

Reply:
@BrianZito this is standard practice for dealing with different calendars. Month names are for calendar months, not fiscal months. It's common to only divide Fiscal years into quarters and stop there anyway.

Instead of thinking by month name, it's more productive to think of the month number.

Examples:
- October is month 10 for calendar year, but it month 1 for federal fiscal year
- April is month 4 for calendar year, month 7 for federal fiscal year, and month 1 for Booz Allen fiscal year